datasets/aardgasverbruik/aardgasverbruik_mra_statistieken_pcranges/v1.0.0.json

```diff
@@ -0,0 +1,58 @@
+{
+  "id": "aardgasverbruik_mra_statistieken_pcranges",
```

Comment:
Deze weer graag in camelcase.

Reply:
Aangepast naar aardgasverbruikMraStatistiekenPcranges